cvat/settings/email_settings.py

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env python
 
 # Copyright (C) 2020-2022 Intel Corporation
+# Copyright (C) 2022 CVAT.ai Corporation
 #
 # SPDX-License-Identifier: MIT
 
 from cvat.settings.production import *
 
 
 # https://github.com/pennersr/django-allauth
-ACCOUNT_AUTHENTICATION_METHOD = 'username'
+ACCOUNT_AUTHENTICATION_METHOD = 'username_email'
```

Comment:
Please, update also `ACCOUNT_AUTHENTICATION_METHOD` in the installation documentation